cardlibrary/10.12.16.lua

```diff
@@ -2527,7 +2550,7 @@ local tentwelvesixteen = { -- CARD_ID, NAME, POWER, HEALTH, RARITY,BIO
 			Name = "Life fam",
 			Description = "Summon two Zombies. Whenever an allied fighter dies, Zabiehunter drains 200 health from your opponent.",
 			["Type"] = "OnSummon",
-			["Power"] = {{"Summon","Zabiehuntertoken"},{"Summon","Zombie"},{"Summon","Zombie"},{"Damage",9999,"Self"}},
+			["Power"] = {{"Summon","Zabiehuntertoken"},{"Damage",9999,"Self"},{"Summon","Zombie"},{"Summon","Zombie"}},
```

Comment:
Wait, when it kills itself before it summons the zombies in the code, can it still summon the zombies?

Reply:
yes, the effect will still go through. it only breaks if said effect has something that affects life after the self destroy effect.